src/common/fail_loc.c

```diff
@@ -141,7 +141,7 @@ daos_fail_init(void)
 	int rc;
 
 	rc = d_fault_inject_init();
-	if (rc)
+	if (rc != 0 && rc != -DER_NOSYS)
```

Comment:
copyright date is way out of date on this file

Reply:
Update as well.